### PR TITLE
Bug 1169353 - Include certified Spark apps on Homescreen. r=daleharvey

### PR DESCRIPTION
**Pull request by DouglasSherk**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0c9b129e90e44a2ca32ad5d971e562b79a60f49f

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=15f7cd092fefa3c73343df03ce289bbdb099395e

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d054a0889517cf02de913aa96ca07ea6cf42cd69

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=058efc158031e7b0dfcace6fd1d58cca92638c9f

File: distros/spark/remote.list [hunk 1]
```diff
@@ -5,10 +5,9 @@ twitter,https://mobile.twitter.com/cache/twitter.webapp
 sechelt,http://sechelt.fxosapps.org/manifest.webapp
 yammer,http://yammer.fxosapps.org/manifest.webapp
 hexgl,http://hexgl.fxosapps.org/manifest.webapp
-foxfooding,https://fxos.github.io/foxfooding/manifest.webapp
```
Comment (DouglasSherk) [inline distros/spark/remote.list]: We're cutting the Foxfooding app, and instead doing email updates to foxfooders.

**Pull request merged**